htdocs/dbmaintain.php

```diff
@@ -5,7 +5,7 @@
  *  Unicode Reminder メモ
  ***************************************************************************/
 
-require __DIR__ . '/lib2/web.inc.php';
+require __DIR__.'/lib2/web.inc.php';
```

Comment:
Up to now, the OC code is pretty consistently using spaces here. I am fine with this change, but think we should add it to our style guides then. When exactly should the spaces be removed? Always, or only when concatenating constants?